rmgpy/data/kinetics/family.py

```diff
@@ -3829,12 +3834,18 @@ def fix_labels_mol(mol, root_labels):
                 for j, react in enumerate(r.item.reactants):
                     if rxns[i].reactants[j].thermo is None:
                         react.generate_resonance_structures()
-                        rxns[i].reactants[j].thermo = tdb.get_thermo_data(react)
+                        if r.metal:
+                            rxns[i].reactants[j].thermo = tdb.get_thermo_data(react, metal=r.metal)
```

Comment:
Doesn't `get_thermo_data` expect` metal_to_scale_to` and/or `metal_to_scale_from` not `metal`?
And if so, clearly this bit of code is not being properly tested so please try to add unit tests as well as fix it.

Also, if the default of `None` acts the same as not defining it, then you don't need the whole `if`/`else` block. Just always call it with the `r.metal` (which is sometimes `None`)?